packages/addresses/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "typeRoots": ["@types", "./node_modules/@types"],
+    "target": "es5",
```

```suggestion
    "target": "es6",
```

I think that's safe enough.